src/js/components/FragmentList/FragmentList.jsx

```diff
@@ -30,7 +30,7 @@ import {
 import FragmentListItem from './FragmentListItem/FragmentListItem';
 import { FRAGMENT_LIST_HEADER, fragmentListTablesHeaders } from '../../helpers/constants';
 
-export function mapDataToComponents(fragments, tabId) {
+export const mapDataToComponents = (fragments, tabId) => {
```

Comment:
Isn't eslint mad here? 
You could have performed a return directly after the arrow:
```
export const mapDataToComponents = (fragments, tabId) => fragments.map(({ debug, nodes }) => {
...
}
```